### PR TITLE
Feature/342/recipe details2

### PR DESCRIPTION
**Pull request by andreaMT15**
### Ticket Number:
342

### Describe The Problem Being Solved:
Fix styling of dropeown menu

### Checklist For Submitter

* [ x] I have documented all new functionality
* [x ] I have screenshotted new UI changes and attached them to this PR
Screen Shot 2019-10-17 at 7 48 47 PM

### CODE REVIEW

**Comment by vhoof** [comment]
I did `git rebase --onto staging HEAD~3` to fix the merge issues, and `git diff HEAD..staging` to make sure that I caught all the relevant changes before force pushing. You'll want to pull your branch down again from Github before you make any further changes (and `git checkout staging && git pull`).

File: frontend/src/components/EditRecipe/EditRecipeForm.css [hunk 1]
```diff
@@ -1,17 +1,24 @@
+textarea, select, input{
```
Comment (vhoof) [inline frontend/src/components/EditRecipe/EditRecipeForm.css]: While we're working on this, we should probably get away from styling every `textarea`, `select`, `input`, and `label` on the entire site. Can we change these (and the ones below) to use classes? If there are site-wide styling decisions we want to make, they should probably go somewhere other than this CSS file.

File: frontend/src/components/EditRecipe/EditRecipeForm.css [hunk 2]
```diff
@@ -49,3 +56,7 @@ label {
   width: 200px;
   border: none;
 }
+
+.btn-container{
```
Comment (vhoof) [inline frontend/src/components/EditRecipe/EditRecipeForm.css]: And maybe something a little more page-specific here.

File: README.md [hunk 3]
```diff
@@ -39,6 +39,22 @@ This project uses [`jest` 23.x](https://jestjs.io/docs/en/23.x/getting-started.h
...
+```javascript
+import map from 'lodash/map';
+```
+
+### Testing
+
+This project uses [`jest` 23.x](https://jestjs.io/docs/en/23.x/getting-started.html) for unit testing. Use the `npm test` command from within the `frontend` folder to run the existing tests.
+
+In order to add new tests for a component (e.g. `YourComponent.js`), add a `YourComponent.test.js` file to the same folder as the component you would like to test.
+
```
Comment (H12) [inline README.md]: This can be removed. It looks like it is being duplicated from above for some reason.

**Pull request merged**